redaxo/src/core/lib/autoload.php

```diff
@@ -86,7 +86,7 @@ public static function autoload($class)
         }
 
         $force = false;
-        $lowerClass = strtolower($class);
+        $lowerClass = strtolower(@utf8_encode($class));
```

Comment:
ich vermute mit so einem fix würde man das gleiche problem für klassen bekommen die UTF8 zeichen enthalten (z.b. mit emoji im klassennamen)

Reply:
You'd better change the class only if it's a non utf8 one. Personnaly, I would even make this more specific and hardcode the special name used by symfony.

Reply:
thanks for your input @nicolas-grekas .

you inspired my to a more acceptable fix. I was not at all convinced from the previous one :-)